### PR TITLE
Testing updates to work with ChefDK

### PR DESCRIPTION
**Pull request by smith**
* Reorder the README to put the license at the end
* Change names of projects it works with
* Remove Gemfile and assume ChefDK
* Update specs to be compatible with new ChefDK/RSpec/ChefSpec

/cc @chef/chef-visual-interaction

### CODE REVIEW

File: spec/recipes/runit_spec.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 describe 'enterprise::runit' do
-  let(:runner) { ChefSpec::Runner.new }
+  let(:runner) { ChefSpec::SoloRunner.new }
   subject(:chef_run) { runner.converge(described_recipe) }
 
   before :each do
     # ChefSpec told me to do this
-    stub_command("grep 'SV:123456:respawn:/opt/opscode/embedded/bin/runsvdir-start' /etc/inittab")
+    stub_command("grep 'SV:123456:respawn:/embedded/bin/runsvdir-start' /etc/inittab")
```
Comment (jtimberman) [inline spec/recipes/runit_spec.rb]: This doesn't seem right to me, don't we enter the /opt/opscode path into the inittab entry?

Comment (smith) [inline spec/recipes/runit_spec.rb]: It was because we were setting the install path after this and it was blank. Fixed in 4e049b7.

**Comment by smith** [comment]
Also cc @chef/lob, I think.

**Comment by jtimberman** [comment]
:+1: Specs pass on my machine.

**Pull request merged**